### PR TITLE
Translation improvements

### PR DESCRIPTION
**Pull request by stejskalleos**
* Extract missing strings for the translations
* Fix interpolations in strings
* Update config for the transifex-cli
* Update *.po files

### CODE REVIEW

**Comment by stejskalleos** [comment]
`foreman_discovery` is broken, it's not failing due to my changes

**Review by ekohl — APPROVED** [review]
Thanks!

**Review by ekohl — DISMISSED** [review]
On second thought, looks like this doesn't update the `.mo` files which are included in the repo.

File: foreman_discovery.gemspec [hunk 1]
```diff
@@ -3,6 +3,12 @@ $:.push File.expand_path("lib", __dir__)
 # Maintain your gem's version:
 require "foreman_discovery/version"
 
+Dir['locale/**/*.po'].each do |po|
+  mo = po.sub(/foreman_discovery\.po$/, 'LC_MESSAGES/foreman_discovery.mo')
+  warn "WARNING: File #{mo} does not exist, generate with 'make all-mo'!" unless File.exist?(mo)
+  warn "WARNING: File #{mo} outdated, regenerate with 'make all-mo'" if File.mtime(po) > File.mtime(mo)
```
Comment (ekohl) [inline foreman_discovery.gemspec]: This causes a build failure:
```
[!] There was an error while loading `foreman_discovery.gemspec`: No such file or directory @ rb_file_s_mtime - locale/cs_CZ/LC_MESSAGES/foreman_discovery.mo. Bundler cannot continue.
```

I think the line above it (`File.exist?`) should stop processing.

**Comment by stejskalleos** [comment]
@adamruzicka review please

File: foreman_discovery.gemspec [hunk 2]
```diff
@@ -3,7 +3,6 @@ $:.push File.expand_path("lib", __dir__)
 # Maintain your gem's version:
 require "foreman_discovery/version"
 
-# Describe your gem and declare its dependencies:
```
Comment (adamruzicka) [inline foreman_discovery.gemspec:6]: Why?

**Review by adamruzicka — APPROVED** [review]
One question, but the changes look sane

**Comment by adamruzicka** [comment]
Also, not sure what's up with the tests

File: foreman_discovery.gemspec (hunk 2 above)
Comment (stejskalleos) [reply foreman_discovery.gemspec:6]: I was doing cleanup and was maybe too hasty. I'll revert the change.

**Comment by stejskalleos** [comment]
Updated. Failing tests are not related, I'm planning to move to github actions anyway, but don't want to block this PR by it.

**Review by adamruzicka — APPROVED** [review]
LGTM

**Review by ares — APPROVED** [review]
lgtm

**Pull request merged**